src/Storage/Plugin/ClearExpiredByFactor.php

```diff
@@ -30,18 +32,18 @@ public function attach(EventManagerInterface $events, $priority = 1)
     /**
      * Clear expired items by factor after writing new item(s)
      *
-     * @param  PostEvent $event
      * @return void
+     * @phpcs:disable Generic.NamingConventions.ConstructorName.OldStyle
```

Comment:
Cannot rename without introducing BC break.

Reply:
Wait, what, we still have this?

Reply:
Yes, but its not used as a `__construct`. Its just a real method which is being invoked from the event manager.